scaladoc/test/dotty/tools/scaladoc/signatures/SignatureTest.scala

```diff
@@ -84,9 +84,11 @@ abstract class SignatureTest(
 
   private def findName(signature: String, kinds: Seq[String]): Option[String] =
     for
-      kindMatch <- kinds.flatMap(k => s"\\b$k\\b".r.findFirstMatchIn(signature)).headOption
+      kindMatch <- kinds.flatMap(k =>s"\\b$k\\b".r.findFirstMatchIn(signature)).headOption
       afterKind <- Option(kindMatch.after(0)) // to filter out nulls
-      nameMatch <- identifierRegex.findFirstMatchIn(afterKind)
+      nameMatch <- identifierRegex.findFirstMatchIn( // The name of an extension will always be the signature itself
```

Comment:
Hmmm, this comment is not true. This line tries to match the first identifier in the signature, so if the signature is e.g. `extension (a: Int)` the first identifier will be `extension`. i.e. every extension signature will have the name `"extension"`. Though I think that the behavior you're describing might be better.
So, either:
1. change the comment
2. or move the if outside of the `findFirstMatchIn` and actually return `Some(signature)` for `extensions`

Reply:
Ho yes, I got confused. Is it better ? I also removed the filterNot in TranslatableSignaturesTestCases because the test pass now.